### ISSUE TITLE
license plate recognition

### ISSUE THREAD

**Issue by masoudnick** [AlexeyAB/darknet#3914]
@AlexeyAB 
Hi,
I want to train a neural network for **license plate recognition** with yolo3, but I don't know to use which Pre-trained model and cfg-file to get best accuracy.
please help me . thanks

![653](https://user-images.githubusercontent.com/20556982/64842562-c76e5780-d617-11e9-893f-61c924cab4a2.jpg)

**Comment by ntd94** [comment]
Have you done the task? Could you share some pieces of information?

**Comment by LukeAI** [comment]
https://github.com/openalpr/openalpr
